src/network/networkd-link.c

```diff
@@ -3097,6 +3104,9 @@ static int link_carrier_lost(Link *link) {
 
         assert(link);
 
+        if (link->setting_mtu == true)
```

Comment:
Drop " == true" everywhere, we don't compare booleans like that.

Reply:
updated